### PR TITLE
Support for tvOS12's new videos

### PR DESCRIPTION
**Pull request by glouel**
This is based on @esetnik fork, modified to download the new tar file : https://sylvan.apple.com/Aerials/resources.tar

It downloads and untar the tar file to the cache directory. Json format slightly changed so accounted for that.

### CODE REVIEW

**Comment by kjg** [comment]
Would any of the changes from his [4k branch](https://github.com/esetnik/Aerial/tree/feature/4k) be useful?

**Comment by glouel** [comment]
Looks like I forked the wrong branch of his... my bad. I'll check his other changes and see what I can do, thanks for the heads up !

**Comment by esetnik** [comment]
@glouel @kjg here's a list of improvements I made

https://github.com/JohnCoates/Aerial/pull/422

File: Readme.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Aerial is completely open source, so feel free to contribute to its development!
 
 ## Installation
 
-1. [Click here to Download](https://github.com/esetnik/Aerial/releases/download/v1.3/Aerial.saver.zip)
+1. [Click here to Download](https://github.com/glouel/Aerial/releases/download/v1.4a1/Aerial.saver.zip)
```
Comment (esetnik) [inline Readme.md]: @glouel we need to be careful here. If we want this to actually get merged into JohnCoates/Aerial then this shouldn't point to the forked repo.

**Comment by glouel** [comment]
I'll apply my changes to your correct branch and resubmit, sorry about that.

File: Readme.md (hunk 1 above)
Comment (kjg) [reply Readme.md]: Is there any hope of any of these changes being pulled back in to this repo here? It doesn't look like it has accepted any real code changes in a long time. Maybe it's time to settle around a new fork?

Comment (esetnik) [reply Readme.md]: I've seen the same thing. I just opened https://github.com/JohnCoates/Aerial/issues/499 to formally ask for more active project maintenance. I think it's best to keep it under the original repo and have active maintainers take over the project. It's already got 11,795 stars and lots of public links to the project on blogs, media, etc.

**Pull request closed without merging**